### PR TITLE
Pull request how to issue 649

### PR DESCRIPTION
**Pull request by architapatelis**
Added directions for how to create a pull request and sync with upstream master to WORKFLOW.md

### CODE REVIEW

File: WORKFLOW.md [hunk 1]
```diff
@@ -53,6 +53,30 @@ You can read about Rubocop [here](http://rubocop.readthedocs.io/en/latest/).
 Make a fork of the repository, create a new branch for every feature you are
 working on! **Always make pull requests!**
 
+### HOW TO: Pull Requests & Syncing
```
Comment (julianguyen) [inline WORKFLOW.md]: Could you add these new sections in our repo's wiki, and then link to it here?

Comment (julianguyen) [reply WORKFLOW.md]: Can you rename this to `Pull Requests & Syncing`?

File: WORKFLOW.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ You can read about Rubocop [here](http://rubocop.readthedocs.io/en/latest/).
...
+
+You can learn more about how to create a pull request and sync your forked repo with the upstream(ifme) repo [here](https://github.com/julianguyen/ifme/wiki/HOW-TO:-Pull-Requests-&-Syncing)
```
Comment (julianguyen) [inline WORKFLOW.md]: Can you add a space between `upstream(ifme)` i.e. `upstream (ifme)`?

Comment (architapatelis) [reply WORKFLOW.md]: @julianguyen sure I can make the changes. But, I wanted to know, do you want the `ifme` in parenthesis or not? i.e do you want `upstream ifme` or `upstream (ifme)`. Also, should I remove the words `HOW TO:` from the title of the [wiki](https://github.com/julianguyen/ifme/wiki/HOW-TO:-Pull-Requests-&-Syncing) as well or just from `WORKFLOW.md`. let me know and I will happily make the changes.

Comment (julianguyen) [reply WORKFLOW.md]: `upstream (ifme)` should be fine!

Yeah let's remove `HOW TO:` from both areas :)

Thanks a bunch!

**Comment by architapatelis** [comment]
@julianguyen done! thanks for the opportunity to make a contribution :)

**Comment by julianguyen** [comment]
Looks like you'll need to rebase from master again! But after that, we're good to go on merging!

**Comment by architapatelis** [comment]
@julianguyen sorry about that, as a student I don't have much experience with merge conflicts and rebasing. But, this was a good learning experience.

**Pull request merged**